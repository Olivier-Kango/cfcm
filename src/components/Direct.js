import React from 'react';

const Direct = () => (
  <div>
    Direct
  </div>
);

export default Direct;
