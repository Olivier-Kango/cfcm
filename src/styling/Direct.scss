.direct {
  padding: 16% 5%;
}
