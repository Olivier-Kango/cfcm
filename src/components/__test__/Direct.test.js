import { render, screen } from '@testing-library/react';
import Direct from '../Direct';

describe('Testing the Direct page', () => {
  test('should render', () => {
    render(<Direct />);
    const about = screen.getByText(/Direct/i);
    expect(about).toMatchSnapshot();
  });
});
