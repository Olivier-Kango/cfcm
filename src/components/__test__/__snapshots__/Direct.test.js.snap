// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the Contact page should render 1`] = `
<div>
  Contact
</div>
`;

exports[`Testing the Direct page should render 1`] = `
<div>
  Direct
</div>
`;
